lib/src/css_class_set.dart

```diff
@@ -41,7 +41,7 @@ abstract class CssClassSet implements Set<String> {
   ///
   /// If [shouldAdd] is true, then we always add that [value] to the element. If
   /// [shouldAdd] is false then we always remove [value] from the element.
-  bool toggle(String value, [bool shouldAdd]);
+  bool toggle(String value, [bool? shouldAdd]);
```

Comment:
Any chance we could have a default value for `shouldAdd`, and make it non-nullable (or is this backed by JS code)?

Reply:
`null` appears to have a meaning that changes depending on other state.

https://github.com/dart-lang/html/blob/d9d0da73abf3c299d42b6000fd72c5b4fefb9162/lib/src/css_class_set.dart#L117